### ISSUE TITLE
CentOs 8 (EOL) is used as a base for images

### ISSUE THREAD

**Issue by MagalyBoddin** [ansible/ansible-runner#1179]
When running the "ansible-builder build" command I end up with an image that is based on CentOs 8, which went EOL in Dec 2021. I'm not sure if this is an issue in ansible-builder or ansible-runner (or both).

Is there any way this can be upgraded to CentOs 9?

**Comment by Akasurde** [comment]
@MagalyBoddin Thanks for reporting this issue. We are right now working on a new Ansible Builder version which will allow developers to use any OS image as a base image and perform more customization options while building it. Once this feature is ready you can build your own images using CentOS 9. You can track the progress of this project here - https://github.com/users/nitzmahone/projects/1.

Meanwhile, you can use https://github.com/ansible/ansible-runner/issues/1152#issuecomment-1304022467 created by https://github.com/ryanmerolle (Thanks!!! Ryan)

Closing this issue as a duplicate of https://github.com/ansible/ansible-runner/issues/1152

**Comment by kurokobo** [comment]
@MagalyBoddin 
F.Y.I., the current base image is based on CentOS _**Stream**_ 8, which is differ from (legacy) CentOS 8 and is not in EOL yet.

```bash
$ docker run --rm quay.io/ansible/ansible-runner:stable-2.12-latest cat /etc/os-release
NAME="CentOS Stream"
VERSION="8"
...
```
